### DISCUSSION TITLE
Windows Store

### DISCUSSION THREAD

**Discussion by TheByteSmasher** [yermak/AudioBookConverter#634] [General]
@yermak Are there any plans to bring the paid version of this app to the Windows/Microsoft store? I have Steam but don't generally load it due to its resource usage. Thanks!

**Comment by yermak** [comment]
Hi @TheByteSmasher,

Unfortunately, Microsoft does not allow disribute applications with own runtime (Java Runtime in case of AudioBookConverter) in Windows/Microsoft store. That's why such option as Steam was selected. At least this was valid when I checked this couple of years back, probably it will be possible with some workarounds - I will try to have a look into this - if there is a quick solution without rewriting application using another language.
